src/data/faq_de.json

```diff
@@ -376,6 +379,217 @@
+                                    "Die aktuellen Isolations-Regeln richten sich nach dem IfSG und nach den lokalen geltenden Bestimmungen."
+                                ]
+                            },
+                            {
+                                "title": "Wenn ich wieder einen negativen Schnelltest habe, kann ich die Isolation dann beenden?",
+                                "anchor": "warn_without_tan_negative",
+                                "active": false,
+                                "textblock": [
+                                    "Die Corona-Warn-App kann keine medizinische Diagnose stellen, ob Sie infiziert sind oder nicht. Daher klären Sie bitte die Frage der Beendigung der Isolation mit dem zuständigen Gesundheitsamt bzw. Ihrem Arzt."
+                                ]
+                            },
+                            {
+                                "title": "Wird das positive Testergebnis bzw. das Warnen in meinem Kontakt-Tagebuch vermerkt?",
+                                "anchor": "warn_without_tan_contact_journal",
+                                "active": false,
+                                "textblock": [
+                                    "Haben Sie das Ergebnis eines PCR-Tests oder Schnelltests mit Hilfe eines QR-Codes oder einer TAN in die Corona-Warn-App eingepflegt, erscheint im Kontakt-Tagebuch ein entsprechender Eintrag.",
+                                    "Wenn Sie manuell über die Funktion \"Tests erfassen und andere warnen\" andere CWA-Anwender*innen direkt warnen, wird kein Test in der App registriert, es wird also auch kein entsprechender Eintrag im Kontakt-Tagebuch erstellt.",
+                                    "Unabhängig von der Art des Tests und wie er registriert wurde, sehen Sie im Kontakt-Tagebuch das Datum der Warnung.",
+                                    "Siehe auch: <a href='#warn_without_tan_how_to' target='_blank'>Wie kann ich andere Nutzer warnen, wenn ich positiv getestet worden bin?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für einen Selbsttest ein offizielles digitales Zertifikat erhalten?",
+                                "anchor": "warn_without_tan_self_test_certificate",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, europaweit sind für Selbsttests keine digitalen Zertifikate vorgesehen. Testzertifikaten können nur für beaufsichtigte PCR-Tests (ausgenommen PoC-NAT PCR-Tests) und beaufsichtigte Schnelltests ausgestellt werden."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für einen registrierten positiven Testbefund als Krankschreibung nutzen?",

```

Comment:
```suggestion
                                "title": "Kann ich einen registrierten positiven Testbefund als Krankschreibung nutzen?",
```

Reply:
Thanks for providing the suggestion for the comment above!